### PR TITLE
chore: update readme + move vega readme under the core directory

### PR DESCRIPTION
**Pull request by jeremyletang**

### CODE REVIEW

**Review by ze97286 — DISMISSED** [review]

File: core/README.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+- [Vega](#vega)
+  - [Links](#links)
+  - [Installation](#installation)
+  - [Configuration](#configuration)
+    - [Files location](#files-location)
+  - [Vega node wallets](#vega-node-wallets)
+    - [Using Ethereum Clef wallet](#using-ethereum-clef-wallet)
+      - [Automatic approvals](#automatic-approvals)
+      - [Importing and generation account](#importing-and-generation-account)
+  - [API](#api)
```
Comment (gordsport) [inline core/README.md:35]: should this be:
`../datanode#data-node` readme content or `./api` directory?

File: core/README.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  - [Provisioning](#provisioning)
```
Comment (gordsport) [inline core/README.md:36]: should this be removed?

File: core/README.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  - [Troubleshooting & debugging](#troubleshooting--debugging)
+
+<!-- tocstop -->
+
+</details>
+
+## Installation
+
+To install `trading-core` and `tendermint`, see [Getting Started](GETTING_STARTED.md).
```
Comment (gordsport) [inline core/README.md:45]:
```suggestion
To install `trading-core` and `tendermint`, see [Getting Started](../GETTING_STARTED.md).
```

File: core/README.md [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+## Troubleshooting & debugging
+
+The application has structured logging capability, the first port of call for a crash is probably the Vega and Tendermint logs which are available on the console if running locally or by journal plus syslog if running on test networks. Default location for log files:
+
+* `/var/log/vega.log`
+* `/var/log/tendermint.log`
+
+Each internal Go package has a logging level that can be set at runtime by configuration. Setting the logging `Level` to `"Debug"` for a package will enable all debugging messages for the package which can be useful when trying to analyse a crash or issue.
+
+Debugging the application locally is also possible with [Delve](./DEBUG_WITH_DLV.md).
```
Comment (gordsport) [inline core/README.md:146]:
```suggestion
Debugging the application locally is also possible with [Delve](../DEBUG_WITH_DLV.md).
```

**Review by gordsport — COMMENTED** [review]
Ive reviewed in the context of the links mainly - the only ones im not sure about are the APIs (should these be core APIs or refer back to data-node?).

File: core/README.md [hunk 5]
```diff
@@ -0,0 +1,146 @@
+# Vega core
+
+A decentralised trading platform that allows pseudo-anonymous trading of derivatives on a blockchain.
+
+**Vega** provides the following core features:
+
+- Join a Vega network as a validator or non-consensus node.
+- [Governance](./governance/README.md) - proposing and voting for new markets
+- A [matching engine](./matching/README.md)
+- [Configure a node](#configuration) (and its [APIs](#apis))
```
[the remark this answers is not in the document]
Comment (jeremyletang) [reply core/README.md:10]: Nope that's fine as is.

File: core/README.md (hunk 1 above)
Comment (jeremyletang) [reply core/README.md:35]: nope this is fine.

**Review by fkondej — APPROVED** [review]

**Pull request merged**